src/documents/en/host/install/index.html.md

```diff
@@ -40,37 +40,45 @@ A minimum amount of RAM is required though: **1024MB** is a good start for 6 Coz
   </a>
 </div>
 
-## on a Virtual Machine
+## on a Virtual Private Server
```

Comment:
`## on a Dedicated Private Server` ? :) (Scaleway C1 servers are not virtualized nor shared)